### PR TITLE
update react native maps

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/components/map.js [hunk 1]
```diff
@@ -81,6 +83,7 @@ export default class Map extends Component {
       buttonTranslateY: new Animated.Value(0),
       userLocationEnabled: false,
       itemLength: 0,
+      itemCategory: '',
```
Comment (yuda1124) [inline shared/components/map.js]: It is already saved in redux. why don't you use state.map.categoryFilter?

Comment (ochanje210) [inline shared/components/map.js]: good point

Comment (ochanje210) [inline shared/components/map.js]: oh no, they are different. categoryFilter is from tabview in main header and this itemCategory is from the marker a user pressed

File: shared/components/map.js [hunk 2]
```diff
@@ -276,8 +287,12 @@ export default class Map extends Component {
     return null;
   }
 
-  renderUserIndicatorMarker() {
-    if (this.props.zoomLevel <= 18 && this.props.zoomLevel > 17) {
+  checkUserZoomLevel() {
+    return (this.props.zoomLevel <= 18 && this.props.zoomLevel > 17);
```
Comment (festinalente91) [inline shared/components/map.js]: isn't this just this.props.zoomLevel == 18 ?

Comment (ochanje210) [inline shared/components/map.js]: no zoomLevel we are having can have floating points. it is actually very hard to get exact zoom level of 18

**Pull request merged**